mac.sh

```diff
@@ -242,66 +194,24 @@ install_postgresql
 brew_install_or_upgrade 'redis'
 brew_launchctl_restart 'redis'
 brew_install_or_upgrade 'sops'
-brew_install_or_upgrade 'the_silver_searcher'
-brew_install_or_upgrade 'vim'
-brew_install_or_upgrade 'ctags'
-brew_install_or_upgrade 'tmux'
-brew_install_or_upgrade 'reattach-to-user-namespace'
-brew_install_or_upgrade 'imagemagick'
 brew_install_or_upgrade 'n'
 sudo n stable
 brew_install_or_upgrade 'yarn'
 brew_install_or_upgrade 'rbenv'
 brew_install_or_upgrade 'ruby-build'
-brew_install_or_upgrade 'wget'
-brew_install_or_upgrade 'autojump'
 brew_install_or_upgrade 'openssl'
 brew unlink openssl && brew link openssl --force
-brew_install_or_upgrade 'z'
 
 brew_tap 'homebrew/cask'
 cask_install_or_upgrade 'chromedriver'
 install_elasticsearch
 
 # Install applications
-cask_install_or_upgrade 'google-chrome'
```

Comment:
I think actually... we might need it for the feature specs.. merp

Reply:
chrome is installed by default on our laptops so I don't think this line is required - but I can add it back anyway

Reply:
oh right! my b, nah leave it out, I forgot that slack + chrome and stuff are installed by IT before we get our laptops now (we used to install it ourselves)